packages/components/input/src/input.vue

```diff
@@ -236,11 +241,23 @@ const inputSize = useFormSize()
 const inputDisabled = useFormDisabled()
 const nsInput = useNamespace('input')
 const nsTextarea = useNamespace('textarea')
+const {
+  wrapperRef,
+  targetRef: input,
+  isFocused: focused,
```

Comment:
There may be no need to assign to the new variable name here, just let `useFocusController` return the corresponding variable name.